docs/openapi3.yaml

```diff
@@ -1365,10 +1411,51 @@ components:
         - dsl
         - basic
         example: dsl
+    bench_mark_response:
```

Comment:
this benchmark resource is not returned anywhere
I think it will be good to return it as part of the test resource
or as GET: /tests/test_id/benchamrk

Reply:
will add it to get in another branch